### PR TITLE
LPS-189366 | Test Fix | Master

### PR DESCRIPTION
**Pull request by kristinonias**
https://liferay.atlassian.net/browse/LPS-189366

cc: @patriciajeanperez

### CODE REVIEW

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -1486,57 +1486,66 @@ definition {
 	test IconOnModalTitleIsGreyWhenItemIsCompletedOnTime {
 		property test.liferay.virtual.instance = "false";
 
-		WorkflowMetrics.goToWorkflowSLASettings(workflowProcessName = "Single Approver");
+		task("Given an SLA with a blog entry") {
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:1489]: Adding task blocks for readability.

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 2]
```diff
@@ -1486,57 +1486,66 @@ definition {
...
 
-		Click(locator1 = "WorkflowMetricsSLA#ADD_NEW_SLA_BUTTON");
+			WorkflowMetrics.executeIndexActions(actionDescription = "Workflow Indexes");
```
Comment (kristinonias) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:1491]: Added the reindex to remove the duplicate Single Approvers.

**Comment by andreefarias** [comment]
ci:test:sf

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**